rdr_service/resource/generators/participant.py

```diff
@@ -947,7 +948,18 @@ def _make_sample_dict_from_row(bss=None, bos=None):
                 logging.error(f'No stored or ordered sample info provided for biobank id {p_bb_id}. Please investigate')
                 return {}
 
+            # Create a unique repeatable primary key value for each biobank sample record.
+            test_id = BIOBANK_UNIQUE_TEST_IDS[test] if test in BIOBANK_UNIQUE_TEST_IDS else '99'
+            if bo_pk:
+                # From known biobank orders
+                id_ = int(f'{bo_pk}{test_id}')
+            else:
+                # For unknown biobank orders, use participant_id + 99 + test id.
+                # All tests here will be grouped together under this 99 id.
+                id_ = int(f'{bss.participant_id}99{test_id}')
```

Comment:
Have we verified if there are any cases where a participant has more than one `biobank_stored_sample` row for the same test type, neither of which has a `biobank_order_id`?  Would that break the key uniqueness?